src/test/java/features/Text Tests.feature

```diff
@@ -14,11 +14,30 @@ Feature: Text Verifcation Tests
   	Then I verify the username field is empty  
   	When I fill the account information into the username field and the password field
   	  And I enter Bob in the username field
-  	  And I randomly enter text in the password field for the fun of it
+  	  And I randomly enter text in the password field
   	  And I reuse the password field text in the password field
   	Then I verify the username field is not empty
   	  And I verify the username field does not contain the text "foo"
   	  And I verify the username field has the text "Bob"
   	When I press the browser back button
   	  And I press the browser forward button
-  	  And I press the browser refresh button
\ No newline at end of file
+  	  And I press the browser refresh button
+
+	@text
+	Scenario: I randomly append/prepend/enter text, and verify the entry
+		Given I am on the Guinea Pig Page
+		When I randomly enter test in the Comments Text Area
+			And I click the Send Button
+		Then I wait until the Your Comments Span contains the text entered for the Comments Text Area
+		When I clear the Comments Text Area
+		  And I uniquely enter test in the Comments Text Area
```

Comment:
the indenting looks off on this scenario compared to the last, as well as on this line specifically. Not sure what's going on there, might be tabs instead of spaces

Reply:
Done! This particular file was fighting it.